python/ray/serve/controller.py

```diff
@@ -404,10 +408,17 @@ def _create_control_loop_metrics(self):
             "serve_controller_sleep_duration_s",
             description="The duration of the last control loop's sleep.",
         )
-        self.control_loop_gauge_s = metrics.Gauge(
+        self.control_loop_duration_gauge_s = metrics.Gauge(
             "serve_controller_control_loop_duration_s",
             description="The duration of the last control loop.",
         )
+        self.num_control_loops_gauge = metrics.Gauge(
+            "serve_controller_num_control_loops",
```

Comment:
can you add the controller's actor_id as a tag on the gauge?

Reply:
I added the `actor_id`.